tests/helpers/run-command.js

```diff
@@ -6,7 +6,9 @@ const defaults = require('ember-cli-lodash-subset').defaults;
 const killCliProcess = require('./kill-cli-process');
 const logOnFailure = require('./log-on-failure');
 let debug = require('heimdalljs-logger')('run-command');
+const captureExit = require('capture-exit');
 
+let RUNS = [];
```

Comment:
I don't understand how this works. AFAICT nothing pushes stuff into `RUNS` (and it is not exported), am I just missing something?

Reply:
we export a helper `killAll` which iterates this structure.

Reply:
good catch!